go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/huandu/xstrings v1.3.0 // indirect
 	github.com/mitchellh/copystructure v1.0.0 // indirect
 	github.com/shurcooL/sanitized_anchor_name v1.0.0 // indirect
+	github.com/spf13/cobra v0.0.5
```

Comment:
Why not the latest version?

Reply:
Not sure.. thats just what I got automatically somehow.

Reply:
Can you try v0.0.7 manually?

Reply:
Done 👍